### PR TITLE
Initial take on vocabulary/tags concept docs

### PR DESCRIPTION
**Pull request by tiagojsag**
This is my initial take on vocabularies and tags. I want to delay the merge of this until I've done some progress on the actual endpoints, as I feel like there's a lot to learn about the concepts by reviewing and documenting about the endpoints.

Still, this is an initial take at explaining both concepts, and while the finer details may improve a bit, the core of it is captured here, so I still think it's worth a review

### CODE REVIEW

**Review by ikas — APPROVED** [review]
Very funky, I like it :ok_hand:

File: source/includes/concept/_vocabulary.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Now that you have your tags in place, your music collection is starting to feel a lot more organized. You notice you can quickly group similar songs, and create playlists like "pop songs" or "95s punk", and you can even use tags to find songs that match your current mood, that you didn't remember you had in your collection. Tags are really powerful to discover resources you didn't know existed - whether it's "pop" songs, "deforestation" datasets, or "social inequality" layers. 
+
+RW API tags are shared by all users, meaning any user can discover your resources if they search by the tags you assigned to them. Similarly, you can use tags to discover resources created and tagged by other users, so id adds discoverability value both ways.
```
Comment (ethanroday) [inline source/includes/concept/_vocabulary.md]:
```suggestion
RW API tags are shared by all users, meaning any user can discover your resources if they search by the tags you assigned to them. Similarly, you can use tags to discover resources created and tagged by other users, so it adds discoverability value both ways.
```

File: source/includes/concept/_vocabulary.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+However, this simple tag mechanism has a few downsides, so you can't simply add individual tags to RW API resources. You need to use vocabularies.
```
Comment (ethanroday) [inline source/includes/concept/_vocabulary.md]: Maybe frame this a little more positively? Like: "However, in order to reach their full potential, tags need to be combined with another concept: vocabularies."

**Review by ethanroday — APPROVED** [review]
I love it! I wish all of our documentation was this playful and fun to read 😄

**Comment by tiagojsag** [comment]
> 
> 
> I love it! I wish all of our documentation was this playful and fun to read 😄

https://www.youtube.com/watch?v=SFGC_YgeQ5w

**Pull request merged**